### PR TITLE
Change add job text depending on hide prop on dashboard

### PR DESCRIPTION
**Pull request by roggervalf**
When clicking on add job, would be nice to change the message of that button, so people would know that by clicking again, the editor will be hidden. This PR tries to handle this.

### CODE REVIEW

**Comment by bradvogel** [comment]
This doesn't seem like something that should be added to arena generically, right? Some folks might want to add jobs in quick succession without the refresh

**Comment by roggervalf** [comment]
> This doesn't seem like something that should be added to arena generically, right? Some folks might want to add jobs in quick succession without the refresh

Thank you @bradvogel I didn't consider that case, I could remove that part, what do you think in the button message change depending on hide prop?

**Comment by bradvogel** [comment]
Yup that looks fine

**Comment by roggervalf** [comment]
Thank you, comments are addressed

File: public/dashboard.js [hunk 1]
```diff
@@ -148,7 +148,12 @@ $(document).ready(() => {
   });
 
   $('.js-toggle-add-job-editor').on('click', function () {
-    $('.jsoneditorx').toggleClass('hide');
+    const addJobText = $('.js-toggle-add-job-editor').text();
+    const shouldHide = addJobText !== 'Add Job';
```
Comment (bradvogel) [inline public/dashboard.js:152]: I can't get this to work. My repro steps:

1. Run the example queue from PR https://github.com/bee-queue/arena/pull/326
image
2. Click the "Add Job" button.

Expected result: the button changes to "Cancel"
Actual result: the button doesn't change:
image

**Review by bradvogel — CHANGES_REQUESTED** [review]

File: public/dashboard.js (hunk 1 above)
Comment (roggervalf) [reply public/dashboard.js]: Thank you @bradvogel, I could try your example and fix this condition, please take a look :eyes:

**Review by bradvogel — APPROVED** [review]
Reviewed and tested.

**Pull request merged**